xsd/netex_framework/netex_responsibility/netex_entity_support.xsd

```diff
@@ -118,19 +118,23 @@ Rail transport, Roads and Road transport
 			<xsd:documentation>Reference to a Type of an ENTITY.</xsd:documentation>
 		</xsd:annotation>
 	</xsd:element>
-	<xsd:complexType name="ClassRefStructure">
+	<xsd:complexType name="ClassRefStructure" mixed="true">
 		<xsd:annotation>
 			<xsd:documentation>Type for a reference to the Class of a ENTITY.</xsd:documentation>
 		</xsd:annotation>
-		<xsd:simpleContent>
-			<xsd:extension base="xsd:string">
-				<xsd:attribute name="nameOfClass" type="NameOfClass" use="required">
-					<xsd:annotation>
-						<xsd:documentation>Name of referenced Class.</xsd:documentation>
-					</xsd:annotation>
-				</xsd:attribute>
-			</xsd:extension>
-		</xsd:simpleContent>
+		<xsd:sequence>
+			<xsd:element name="MemberRef" type="ClassRefStructure" minOccurs="0" maxOccurs="unbounded"/>
+		</xsd:sequence>
+		<xsd:attribute name="nameOfClass" type="NameOfClass" use="required">
+			<xsd:annotation>
+				<xsd:documentation>Name of referenced Class.</xsd:documentation>
+			</xsd:annotation>
+		</xsd:attribute>
+		<xsd:attribute name="use" type="MandatoryEnumeration" default="optional">
```

Comment:
Where is MandatoryEnumeration defined now?

Reply:
In the same file.

Reply:
xsd\netex_framework\netex_responsibility\netex_entity_support.xsd

Reply:
This was previously already in?